### PR TITLE
poppler:update to 24.10.0

### LINKED ISSUE #22328: [poppler] update to 24.10.
### Package name

poppler

### New version number

24.10.0

### Other information that may be useful

I have updated and run PKGBUILD in my fork.
The problem is that inkscape does not build anymore as poppler uses now smartpointers for some functions.
What is the policy here ?

### Are you willing to submit a PR?

yes

### PR DESCRIPTION
**Pull request by sunderme**
* Fixes #22328

### CODE REVIEW

**Comment by Biswa96** [comment]
Inkscape has merged the fix for newer poppler. Also, there is a new version of poppler. I am not sure if it is compatible with Inkscape yet.

**Comment by Biswa96** [comment]
Just curious, is it required to update poppler so frequently? poppler breaks things with every release and all the reverse dependencies required to be rebuilt.

**Comment by sunderme** [comment]
poppler updates every month and more often than not, they talk about "fix crash/memory handling".
I have not had any user reports on poppler issues recently but after all, it is used to handle data from the internet ...

File: mingw-w64-inkscape/PKGBUILD [hunk 1]
```diff
@@ -6,7 +6,7 @@ _realname=inkscape
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=1.4
-pkgrel=1
+pkgrel=3
```
Comment (Biswa96) [inline mingw-w64-inkscape/PKGBUILD:9]:
```suggestion
pkgrel=2
```

Comment (sunderme) [reply mingw-w64-inkscape/PKGBUILD:9]: it is 
`
pkgver=1.4
pkgrel=2
` in master ?

Comment (Biswa96) [reply mingw-w64-inkscape/PKGBUILD:9]: Oh, I did not notice it. Then, it's ok.

**Comment by sunderme** [comment]
now gdcm fails

**Comment by sunderme** [comment]
and another bug report: https://gitlab.freedesktop.org/poppler/poppler/-/issues/1537

**Comment by sunderme** [comment]
and now inkscape/clang64 fails to build, i have included  the official patch

**Comment by sunderme** [comment]
looks like c++20 & pdf2djvu issue ?

**Comment by lazka** [comment]
> Since pdf2djvu is out of maintenance for 2 years, this is out of my scope.

I've dropped the clang builds in 014a8c2ae0157625f3867874bfce6f224c8ad152

**Comment by lazka** [comment]
dia rebuild is missing, I'll push that separately

**Pull request merged**